src/Components/test/testassets/BasicTestApp/Index.razor

```diff
@@ -60,13 +60,17 @@
         <option value="BasicTestApp.DuplicateAttributesComponent">Duplicate attributes</option>
         <option value="BasicTestApp.MovingCheckboxesComponent">Moving checkboxes diff case</option>
         <option value="BasicTestApp.LaggyTypingComponent">Laggy typing</option>
+        <option value="BasicTestApp.CulturePicker">Culture Picker</option>
+        <option value="BasicTestApp.LocalizedText">Localized Text</option>
     </select>
 
-  @if (SelectedComponentType != null)
-  {
-    <span id="source-info"><code><tt>@(SelectedComponentType.Name.Replace(".", "/")).cshtml</tt></code></span>
-  }
-  <hr />
+    <span id="runtime-info"><code><tt>@System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription</tt></code></span>
```

Comment:
This now shows you if you're running on mono or coreclr. This seemed like a nice improvement to me since I always confuse myself when playing with the test apps.